### PR TITLE
fix(gen2): Cognito Auth overrides update

### PR DESCRIPTION
**Pull request by ErikCH**
Added Cognito override auth section

### CODE REVIEW

File: src/pages/gen2/build-a-backend/auth/enable-sign-up/index.mdx [hunk 1]
```diff
@@ -187,6 +187,25 @@ const { cfnUserPool } = backend.resources.auth.resources.cfnResources;
 cfnUserPool.addPropertyOverride('Policies.PasswordPolicy.MinimumLength', 32);
 ```
 
+## Customize Cognito UserPool email verification settings
+
+You can change the verification email by overriding the `verificationEmailStyle` and the `verificationEmailBody`.
+
+```ts title="amplify/auth/resource.ts"
```
Comment (josefaidt) [inline src/pages/gen2/build-a-backend/auth/enable-sign-up/index.mdx:194]:
```suggestion
```ts title="amplify/auth/resource.ts"
// amplify/auth/resource.ts
```

File: src/pages/gen2/build-a-backend/auth/override-cognito/index.mdx [hunk 2]
```diff
@@ -27,7 +27,40 @@ export const auth = defineAuth({
...
+    PasswordPolicy: {
+      MinimumLength: 10,
+      RequireLowercase: true,
+      RequireNumbers: true,
+      RequireSymbols: true,
+      RequireUppercase: true,
+      TemporaryPasswordValidityDays: 20,
+    },
+  }
+);
```
Comment (josefaidt) [inline src/pages/gen2/build-a-backend/auth/override-cognito/index.mdx:56]:
```suggestion
);
```
```

Comment (josefaidt) [reply src/pages/gen2/build-a-backend/auth/override-cognito/index.mdx:56]: I'm not sure if that suggestion is going through 😅 but this example is missing its closing backticks

File: src/pages/gen2/build-a-backend/auth/override-cognito/index.mdx [hunk 3]
```diff
@@ -40,9 +73,11 @@ const backend = defineBackend({
 // extract L1 CfnUserPool resources
 const { cfnUserPool } = backend.resources.auth.resources.cfnResources;
 // use CDK's `addPropertyOverride` to modify properties directly
-cfnUserPool.addPropertyOverride('Policies.PasswordPolicy.MinimumLength', 32);
+cfnUserPool.addPropertyOverride("Schema", [
```
Comment (josefaidt) [inline src/pages/gen2/build-a-backend/auth/override-cognito/index.mdx:76]: does this overwrite the existing schema (with `email`)?

File: src/pages/gen2/build-a-backend/auth/override-cognito/index.mdx (hunk 2 above)
Comment (ErikCH) [reply src/pages/gen2/build-a-backend/auth/override-cognito/index.mdx:56]: OK, I added back the backticks

File: src/pages/gen2/build-a-backend/auth/enable-sign-up/index.mdx (hunk 1 above)
Comment (ErikCH) [reply src/pages/gen2/build-a-backend/auth/enable-sign-up/index.mdx:194]: I think this comment is just backticks, I added it back in.

File: src/pages/gen2/build-a-backend/auth/override-cognito/index.mdx (hunk 3 above)
Comment (ErikCH) [reply src/pages/gen2/build-a-backend/auth/override-cognito/index.mdx:77]: No, it just adds a custom attribute.

File: src/pages/gen2/build-a-backend/auth/enable-sign-up/index.mdx (hunk 1 above)
Comment (josefaidt) [reply src/pages/gen2/build-a-backend/auth/enable-sign-up/index.mdx:194]: this one is to add a comment at the top of the snippet with a path to the file

Comment (ErikCH) [reply src/pages/gen2/build-a-backend/auth/enable-sign-up/index.mdx:194]: Ok will do

Comment (ErikCH) [reply src/pages/gen2/build-a-backend/auth/enable-sign-up/index.mdx:194]: #[c99ffe9](https://github.com/aws-amplify/docs/pull/6516/commits/c99ffe9a7b05513136340406bad22d05b888d5bf)

**Review by josefaidt — APPROVED** [review]

**Review by jacoblogan — APPROVED** [review]

**Pull request merged**